setuptools/independent/DepsProcessor/project.json

```diff
@@ -16,6 +16,7 @@
   "frameworks": {
     "netcoreapp1.0": {
       "imports": [
+        "netstandard1.5",
```

Comment:
This is unncessary.  `netcoreapp1.0` already references `netstandard1.5`.  You don't need to import it explicitly.